src/components/AddComment.jsx

```diff
@@ -0,0 +1,85 @@
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
+import { useContext, useState, useEffect } from "react";
+import { CommentsContext } from "../contexts/CommentsContext";
+import { postComment } from "../api";
+
+export const AddComment = ({ article_id }) => {
+  const [commentInput, setCommentInput] = useState("");
+  const { setComments } = useContext(CommentsContext);
+  const [uploadMessage, setUploadMessage] = useState(null);
```

Comment:
A general React tip for State and avoiding bugs 🐛 

Try to always keep a state the same data type, for example your state `uploadMessage` is defaulting as `null` but later being set as a `string`. This has the potential to cause typeErrors in future development if you were to use a string method on the state, it's safer to have a starting value of `''` empty string 👍